Tests/StreamChatTests/StreamChatStressTestPlan.xctestplan

```diff
@@ -12,10 +12,11 @@
     "codeCoverage" : false,
     "environmentVariableEntries" : [
       {
-        "key" : "STRESS_TESTS",
-        "value" : "TRUE"
+        "key" : "TEST_INVOCATIONS",
+        "value" : "10"
```

Comment:
super nit: `version` property somewhere below is `1` . This value should support Integers AFAIK...

Reply:
Well it does not, as [environment variables](https://developer.apple.com/documentation/foundation/processinfo/1417911-environment) are always `String`

Reply:
Wow now on second look I saw the dict in the file... Thanks :)